### PR TITLE
update README according to #737

### PR DESCRIPTION
**Pull request by patarapolw**
According to https://github.com/umputun/remark42/issues/737, actually.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,11 +562,14 @@ To bring it up run:
...
-docker-compose -f compose-dev-backend.yml up
+cp compose-dev-backend.yml compose-private.yml
 # if you mainly work on frontend
-docker-compose -f compose-dev-frontend.yml build
-docker-compose -f compose-dev-frontend.yml up
+cp compose-dev-frontend.yml compose-private.yml
+# now, edit / debug `compose-private.yml` to your heart's content.
+
+# build and run
+docker-compose -f compose-private.yml build
```
Comment (paskal) [inline README.md:571]: I would propose to leave `docker-compose build` commands near `cp` instead of moving to separate section. Otherwise, make sense!

**Comment by paskal** [comment]
I thought about it some more and now I see that there might be some misunderstanding which you might help to figure out using the documentation.

The section you are changing is supposed to run as-is, with generic compose files. If you want to customize docker-compose file and run your own development code on top of it, you should copy it to `compose-private.yml` and run `make rundev` in order to build new dev image and run remark42 locally from it. Details are in the makefile:

```make
backend:
	docker-compose -f compose-dev-backend.yml build

frontend:
	docker-compose -f compose-dev-frontend.yml build

rundev:
	docker pull umputun/baseimage:buildgo-latest
	SKIP_BACKEND_TEST=true SKIP_FRONTEND_TEST=true docker-compose -f compose-private.yml build
	docker-compose -f compose-private.yml up
```

Now I see that makefile is not reflected in the Readme at all, It would be awesome if you'll test it and update the instruction in a way you understand it. Thank you for raising it!

**Comment by patarapolw** [comment]
I would want to somehow edit `docker-compose.yml`, without worrying about tampering with PR. That's why I prefer gitignored file.

I understand about `make rundev`. Seems to have things I need. However, I am not sure how to add to `README.md`...

**Comment by paskal** [comment]
That's how frontend development described currently:

```shell
cp compose-dev-frontend.yml compose-private-frontend.yml
docker-compose -f compose-private-frontend.yml up --build
cd frontend
npm run dev
```

I think, similar section with `make rundev` might be added to backend development guide section. And the part which you changed now could potentially be left unaltered or not, based on how you see it fit the whole document and new section for the backend.

**Comment by vdimir** [comment]
Also maybe adding `compose-private.yml` to `.gitignore` is an option?

**Comment by umputun** [comment]
it is in .gitignore already, unless I missed your point https://github.com/umputun/remark42/blob/master/.gitignore#L25

**Comment by vdimir** [comment]
> it is in .gitignore already, unless I missed your point https://github.com/umputun/remark42/blob/master/.gitignore#L25

Yes, sorry, I haven't check file before writing comment.

**Pull request merged**